NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -51,36 +51,20 @@ const IVStyle style =
 
 NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
 : Plugin(info, MakeConfig(kNumParams, kNumPresets))
-, mInputPointers(nullptr)
```

Comment:
[Sorry for incomplete review, short on time] isn't it potentially problematic not to initialize these?

Reply:
mInputPointers is initialised inline instead